tests/integration/sample_test.go

```diff
@@ -0,0 +1,18 @@
+// +build integration
+
+package integration
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Topic_CreateTopic(t *testing.T) {
+	url := fmt.Sprintf("http://%s:8082/v1/healthcheck", os.Getenv("METRO_TEST_HOST"))
```

Comment:
if all components are running, how do we know which component is running on which port on local, should we read it from config somehow

Reply:
we'll have to parse config files for this in the test case. I guess we can live by convention?